### ISSUE TITLE
Make MD019 configurable for accepting indentations in multiples of 4

### ISSUE THREAD

**Issue by Bengt** [markdownlint/markdownlint#400]
## Describe the Enhancement:

I would like to make the indentations of my ATX-style header a multiple of 4.

### Impacted Rules:

-   MD019

## Describe the Need:

Users like myself like to indent their headings to the first tab stop of 4 spaces, because that aligns nicely with list and code block indentations and thus looks nicer:

```markdown
#   Title Heading
##  Section Heading
### Subsection Heading
```

Note that this also enables setting the indentation to 4 in the editor and avoid adding the correct number of spaces manually, which eliminates a source of errors.

Also, it reminds us of the indentation level we are currently writing at. `README.md` files should really not need more than 3 levels of indentations, so these are enough. If you go beyond three levels of indentations, another tab stop gives you a visually jarring lot of space which can be considered a visual error message:

```markdown
### Subsection Heading
####    Subsubsection Heading
```

## Current Alternative

Currently, the only way to have headings formatted to a multiple of 4 in a file checked by is to disable the rule MD019.

`.mdlrc-style.rb`:

```ruby
exclude_rule 'MD019'
```

## Can We Help You Implement This?:

I do not know any Ruby, so someone else please implement this.

**Comment by jaymzh** [comment]
This isn't a common practice, and I personally dislike vertical alignment in text files. I have no problem with such a feature if someone wants to implement it, but unless you or someone wants to implement it, I don't expect to get around to this given the long backlog. So I'll close, but if someone chimes in they want to implement, we can re-open.

**Comment by Bengt** [comment]
@jaymzh I see. Thanks for considering this.
